qiita_db/metadata_template.py

```diff
@@ -1051,6 +1085,27 @@ def get_filepaths(self, conn_handler=None):
 
         return [(fpid, base_fp(fp)) for fpid, fp in filepath_ids]
 
+    def categories(self):
+        """Get the categories associated with self
+
+        Returns
+        -------
+        set
+            The set of categories associated with self
+        """
+        conn_handler = SQLConnectionHandler()
+        table_name = self._table_name(self.study_id)
+
+        raw = conn_handler.execute_fetchall("""
+            SELECT column_name
+            FROM information_schema.columns
+            WHERE table_name='{0}'""".format(table_name))
```

Comment:
similar comment to above regarding verifying `table_schema` and `table_catalog` are as expected (equal to `'qiita'`)